### DISCUSSION TITLE
Problem Understanding Accessibility

### DISCUSSION THREAD

**Discussion by avwerosuotrust** [community/community#32766] [Accessibility]
Hello Guys 

I have a basic knowledge of accessibility
But am having trouble understanding 
How to make my website Keyboard friendly, can anyone please explain how i can build website that supports keyboard navigation for me

**Comment by benjiallen** [comment]
I like this resource: [Keyboard accessibility: How to test](https://accessibility-manual.dwp.gov.uk/tools-and-resources/basic-accessibility-checks/3-keyboard-accessibility-how-to-test).

Deque has a tool, which you have to pay for, which helps with keyboard testing. [Axe™ Updates: Introducing the Keyboard Guided Test](https://www.deque.com/blog/axe-updates-introducing-the-keyboard-guided-test/).

Deque also has [Deque University](https://dequeuniversity.com/) which has lots of good content on keyboard accessibility. [WebAim also has good articles on keyboard accessibility](https://webaim.org/techniques/keyboard/).

**Reply by avwerosuotrust** [reply]
Thanks for the info

**Comment by cehfisher** [comment]
Hi @avwerosuotrust - this issue appears to be resolved. Could you please confirm and close the issue? Thank you! (cc @queenofcorgis)
